intro/classes/index.html

```diff
@@ -185,19 +174,168 @@ <h3>Single leading underscores</h3>
+            would be in the case of more complicated inheritance structures (long chains of inheritance or multiple inheritance,
+            both of which are well outside the scope of this article and game design in general, but a link is at the bottom of the
+            page for an excellent video on the topic from the 
+            <a href="https://www.youtube.com/c/mCodingWithJamesMurphy" class="new-tab">mCoding</a> YouTube channel).
+        </p>
+
+        <p>
+            The power of class inheritance is that attributes are <i>inherited</i> from the superclass to the subclass. So, an instance of
+            <code>NPC</code> as given above has the same attributes as <code>Actor</code> (note that this would not be the case if we had
+            not called the constructor of the superclass). Here's an example to play around with:
+            
+            <iframe width='800px' height='200px' class="python-embed" src="https://godbolt.org/e#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:python,selection:(endColumn:18,endLineNumber:18,positionColumn:18,positionLineNumber:18,selectionStartColumn:18,selectionStartLineNumber:18,startColumn:18,startLineNumber:18),source:'class+Actor:%0A++++def+__init__(self,+position,+image,+size):%0A++++++++self.image+%3D+image%0A++++++++self.size+%3D+size%0A++++++++self.pos+%3D+position%0A++++%0A++++def+talk(self,+text):%0A++++++++print(text)%0A%0Aclass+NPC(Actor):%0A++++def+__init__(self,+position,+image,+size,+dialogue):%0A++++++++++++super().__init__(position,+image,+size)%0A++++++++++++self.dialogue+%3D+dialogue%0A++++%0Anpc+%3D+NPC((100,+200),+%22assets/npc.png%22,+(20,+20),+%22Hello+World!!%22)%0Aprint(npc.pos)%0Aprint(npc.dialogue)%0Anpc.talk(%22Hello%22)'),l:'5',n:'0',o:'Python+source+%231',t:'0')),k:50,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:executor,i:(argsPanelShown:'1',compilationPanelShown:'0',compiler:python310,compilerOutShown:'0',execArgs:'',execStdin:'',fontScale:14,fontUsePx:'0',j:1,lang:python,libs:!(),options:'',source:1,stdinPanelShown:'1',tree:'1',wrap:'1'),l:'5',n:'0',o:'Executor+Python+3.10+(Python,+Editor+%231)',t:'0')),k:50,l:'4',n:'0',o:'',s:0,t:'0')),l:'2',n:'0',o:'',t:'0')),version:4"></iframe>            But remember from the beginning what I said about attributes and methods? I said that methods <b>are</b> attributes, so 
+            anything that applies to variable attributes also applies to methods! This means that methods are inherited too. Try it
+            in the embed above. Define a method of the <code>Actor</code> class and try to call it from the instance of <code>NPC</code>.
+            One example of that might look like:
+
+{% highlight python linenos %}
+class Actor:
+    def __init__(self, position, image, size):
+        self.image = image
+        self.size = size
+        self.pos = position
+    
+    def talk(self, text):
+        print(text)
+
```

Comment:
Should have 2 blank lines between classes per PEP8

Reply:
This embed still needs two blank lines between class definition and creating those variables (in case I have pointed to the wrong embed, well, one of the embeds needs that)

Ah, maybe I don't see that other embed in the diff... but here
![image](https://user-images.githubusercontent.com/83066658/210113262-3a0309c8-45a9-4fa1-baee-f2c4deb320c4.png)